<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThingAdd</class>
 <widget class="QDialog" name="ThingAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Thing_Confirm">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Thing_Cancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>191</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>物品名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Thing_Name">
        <property name="placeholderText">
         <string>物品名称</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>起拍价格</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Thing_money">
        <property name="placeholderText">
         <string>起拍价格</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>新旧程度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Thing_New">
        <property name="placeholderText">
         <string>以100为全新</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Thing_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>ThingAdd</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
